packages/charting/src/components/Legends/examples/LegendsOverflowExample.tsx

```diff
@@ -11,118 +11,169 @@ export class LegendOverflowExample extends React.Component<{}, {}> {
         action: () => {
           console.log('click from LegendsPage');
           alert('Legend1 clicked');
+        },
```

Comment:
Nit: can these example hoverAction functions include the name of the legend so that it's clear to the user which legend is being hovered? E.g. `console.log('Hover action on Legend1')`.